index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>Portfolio</title>
+</head>
+<body>
+   
+        <!---Header-->
+        <header class="tool-bar">
+            <a href="#">Welcome</a>
+            <nav class="menu"><i class="fas fa-bars"></i></nav>
+        </header>
+           
+        <div class="container">
+            
+            <h2>
+                Hey There.<br>
+                I'm Somdotta
+            </h2>
```

Comment:
- [ ] As the other code reviewer said, please consider changing the `h2` tag for an `h1` tag, this will make your code more semantic and accessible for screen readers. 👍🏻

Reply:
I have done in another branch forget to merge with feature branch.